packages/@ourworldindata/grapher/src/scatterCharts/ScatterPlotChart.tsx

```diff
@@ -809,40 +784,98 @@ export class ScatterPlotChart
                         </g>
                     </>
                 )}
-                {target && (
-                    <Tooltip
-                        id="scatterTooltip"
-                        tooltipManager={this.manager}
-                        x={position.x}
-                        y={position.y}
-                        offsetX={20}
-                        offsetY={-16}
-                        style={{ maxWidth: "250px" }}
-                        title={target.series.label}
-                        subtitle={timeLabel}
-                        dissolve={fading}
-                        notice={targetNotice}
-                    >
-                        <TooltipValueRange
-                            column={this.xColumn}
-                            values={values.map((v) => v.x)}
-                            notice={xNotice}
-                        />
-                        <TooltipValueRange
-                            column={this.yColumn}
-                            values={values.map((v) => v.y)}
-                            notice={yNotice}
-                        />
-                        <TooltipValueRange
-                            column={this.sizeColumn}
-                            values={compact(values.map((v) => v.size))}
-                        />
-                    </Tooltip>
-                )}
+                {this.tooltip}
             </g>
         )
     }
 
+    @computed get tooltip(): JSX.Element | null {
+        if (!this.tooltipState.target) return null
+
+        const {
+            xColumn,
+            yColumn,
+            tooltipState: { target, position, fading },
+        } = this
+        const points = target.series.points ?? []
+        const values = compact(uniq([first(points), last(points)]))
```

Comment:
Same comment about `compact` applies here as it does in `LineChart.tsx`: There are several uses on numbers here that will wrongly exclude 0, please change that to something else.